index.html

```diff
@@ -1615,18 +1615,57 @@ <h3>Media Types</h3>
+			The Web of Things provides specific building blocks to allow the implementation of systems that
+                        conform with the abstract WoT Architecture.
+			The specfics of these building blocks are defined in separate documents; 
+                        this document provides an overview and a summary.
 		</p>
-		
-		<p>
 			<figure id="arch-new-building-blocks">
-				<img src="images/arch-new-building-blocks.png" />
-				<figcaption>Building Blocks</figcaption>
+				<img src="images/wot-building-blocks.png" />
+				<figcaption>Architectural Aspects of a Thing</figcaption>
+			</figure>
+		<p>
+                        The use cases and the requirements discussed result in an abstract model of 
+                        a Thing that has four architectural aspects of interest: its behavior, its interaction affordances,
+                        the mapping of those affordances to concrete protocols (what we will call the protocol binding),
+                        and its security configuration.  
+                        The behavior aspect includes both lifecycle management (onboarding, updating, decommissioning, etc) but
+                        also the operational behavior of the Thing, which can include both autonomous behavior and support 
+                        for the affordances.  The interaction affordances are an abstract model of the network API in terms
+                        of properties, actions, and events, but without reference to a specific network protocol or
+                        payload encoding.  The protocol binding adds the additional detail needed to map each
+                        affordance to a particular concrete protocol; not that different affordances may be supported
+                        by different protocols.  The security configuration aspect represents the mechanisms used to
+                        control access to the affordances and the management of related public and private metadata.
+                        <a href="#arch-new-building-blocks"></a> shows these aspects diagrammatically.
+                </p>
+			<figure id="arch-thing-with-scripting">
+				<img src="images/wot-thing-with-scripting.png" />
+				<figcaption>Thing with WoT Building Blocks, including the WoT Scripting API</figcaption>
+			</figure>
+			<figure id="arch-thing-without-scripting">
+				<img src="images/wot-thing-without-scripting.png" />
+				<figcaption>Thing with WoT Building Blocks, without the WoT Scripting API</figcaption>
 			</figure>
-			As already discussed, consideration of the use cases and the requirements derived therefrom resulted
-			in noting that there are three types of entities. The figure shows the functional architecture that 
-			commonly applies to those three entity types.
+                <p>
+                        The WoT Building Blocks support each of these aspects, as shown in  
```

Comment:
Please consider moving the text to a place above the the figures, so the links become "forward references"

Reply:
figures generally moved to after text that refer to them.  Done.